docker-compose.yml

```diff
@@ -720,7 +741,7 @@ services:
 
   openruntimes-executor:
     container_name: openruntimes-executor
-    hostname: appwrite-executor
+    hostname: exc1
```

Comment:
The .env file should have needed a change too, right?

Reply:
Corrected, .env file did not get pushed in my commit